posthog/models/experiment.py

```diff
@@ -30,6 +30,8 @@ class ExperimentType(models.TextChoices):
     created_by = models.ForeignKey("User", on_delete=models.SET_NULL, null=True)
     feature_flag = models.ForeignKey("FeatureFlag", blank=False, on_delete=models.RESTRICT)
     exposure_cohort = models.ForeignKey("Cohort", on_delete=models.SET_NULL, null=True)
+    holdout = models.ForeignKey("ExperimentHoldout", on_delete=models.SET_NULL, null=True)
```

Comment:
This references the entire model in the backend, but only returns the ID to the frontend (is this a serializer thing?).

Could we return the entire holdout instance in `experiment.holdout` like we do for e.g `experiment.feature_flag`? Would be less confusing and could grab the holdout name for the UI directly from there.

Reply:
done, indeed a serializer thing

Reply:
okay let's merge this